### ISSUE TITLE
OS.set_clipboard() doesn't work on iOS

### ISSUE THREAD

**Issue by kyoz** [godotengine/godot#52228]
### Godot version

3.3.2-stable

### System information

Macbook Pro 15" Mid 2015 - BigSur 11.5.2

### Issue description

Maybe i'm missing something but this doesn't work on iOS, both on Simulator (14.0.1) and Real Device (Iphone 5s - 12.0.1)

```
OS.set_clipboard("Hello")

or

OS.clipboard = "Hello"
```

They work well on Android but doesn't know why it doesn't work on iOS. There is no error when calling too so i don't know exactly it's an error or missing something.

### Steps to reproduce

Just calling these on an iOS simulators or real devices.

```
OS.set_clipboard("Hello")

or

OS.clipboard = "Hello"
```

### Minimal reproduction project

[iOS-Clipboard-issue.zip](https://github.com/godotengine/godot/files/7072839/iOS-Clipboard-issue.zip)

**Comment by Calinou** [comment]
Does iOS even support setting clipboard data outside of its virtual keyboard?

**Comment by kyoz** [comment]
Hi @Calinou , iOS does have [UIPasteBoard](https://developer.apple.com/documentation/uikit/uipasteboard)

By global search on main branch i can see [UIPasteBoard implement for ios](https://github.com/godotengine/godot/blob/master/platform/iphone/display_server_iphone.mm#L546-L554)

But in 3.3.2-stable i can't find any thing relate to UIPasteBoard

So i manually add these and it work :D. Doesn't know if it correctly or i'm missing something but for now it work for me.

platform/iphone/os_iphone.h
```
  virtual void set_clipboard(const String &p_text) ;
  virtual String get_clipboard() const;
```

platform/iphone/os_iphone.mm
```
void OSIPhone::set_clipboard(const String &p_text) {
	[UIPasteboard generalPasteboard].string = [NSString stringWithUTF8String:p_text.utf8()];
}

String OSIPhone::get_clipboard() const {
	NSString *text = [UIPasteboard generalPasteboard].string;

	return String::utf8([text UTF8String]);
}
```

**Comment by Calinou** [comment]
> So i manually add these and it work :D. Doesn't know if it correctly or i'm missing something but for now it work for me.

Feel free to open a pull request for this against the `3.x` branch :slightly_smiling_face:

**Comment by kyoz** [comment]
Ok @Calinou

**Comment by akien-mga** [comment]
Fixed by #52540.
